src/main/resources/public/jsx/review-institution.js

```diff
@@ -46,7 +46,7 @@ class ReviewInstitution extends React.Component {
                     documentRoot={this.props.documentRoot}
                     institutionId={this.props.institutionId}
                     isAdmin={this.state.isAdmin}
-                    of_users_api_url={this.props.of_users_api_url}
```

Comment:
this is passed from another part of the application. `ofUsersApiUrl={this.props.of_users_api_url}`